### PR TITLE
standard and travis

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: test/nested.js [hunk 1]
```diff
@@ -54,9 +53,9 @@ tape('nested encode + decode', function(t) {
       num: 2,
       payload: new Buffer('bar')
     }
-  });
+  })
 
-  var o2 = Nested.decode(b1)
+  var o2 = Nested.decode(b2)
```
Comment (mafintosh) [inline test/nested.js]: nice catch

Comment (ralphtheninja) [inline test/nested.js]: `standard` caught it, I just followed orders! ;)

Comment (mafintosh) [inline test/nested.js]: lol. so many of these as well!

**Comment by mafintosh** [comment]
Released in 3.1.5

**Pull request merged**